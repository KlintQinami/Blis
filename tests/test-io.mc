int main()
{
  u8[18] path = "./tests/test-io.mc";
  u8[] fpath = u8[](length(path));
  int i;
  for (i = 0; i < length(path); i = i + 1)
    fpath[i] = path[i];
  u8[] file = read_file(fpath);
  print(file);
  return 0;
}
