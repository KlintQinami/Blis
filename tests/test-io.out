int main()
{
  u8[] file = read_file("./tests/test-io.mc");
  print(file);
  return 0;
}
